packages/material-ui/src/Typography/Typography.d.ts

```diff
@@ -1,13 +1,15 @@
 import * as React from 'react';
-import { StandardProps, PropTypes } from '..';
-import { OverrideProps, OverridableTypeMap, OverridableComponent } from '../OverridableComponent';
+import { PropTypes } from '..';
+import { OverrideProps, OverridableComponent } from '../OverridableComponent';
 import { Variant as ThemeVariant } from '../styles/createTypography';
 
-type Variant = ThemeVariant | 'srOnly';
+export type TypographyVariant = ThemeVariant | 'srOnly';
+
+export type TypographyAlignment = PropTypes.Alignment;
 
 export interface TypographyTypeMap<P = {}, D extends React.ElementType = 'span'> {
   props: P & {
-    align?: PropTypes.Alignment;
+    align?: TypographyAlignment;
```

Comment:
We don't need this as explained in https://github.com/mui-org/material-ui/issues/19572#issuecomment-583023754

Reply:
Gotcha, will do

Reply:
@eps1lon fixed